src/graph.rs

```diff
@@ -402,7 +402,7 @@ impl PyGraph {
     /// Return a list of all edge data.
     ///
     /// :returns: A list of all the edge data objects in the graph
-    /// :rtype: list
+    /// :rtype: list[Any]
```

Comment:
It should probably say `list[T] where T is the edge type of PyGraph[S, T]` . I will also try to add `(see typing for PyGraph)` in a future PR explaining how to use static type checking with `rustworkx`